queries/special.py

```diff
@@ -1136,13 +1178,17 @@ def _play_music(qs, q):
 
     # Gender, self-identity
     "ertu kona": _COMPUTER_PROGRAM,
+    "ertu kvenmaður": _COMPUTER_PROGRAM,
```

Comment:
Maybe the answer should indicate that the user can choose a male or female voice?
E.g., "Þú mátt ráða hvort ég hef karlrödd eða kvenrödd."